src/main/java/io/aiven/kafka/connect/common/output/jsonwriter/KeyBuilder.java

```diff
@@ -25,12 +25,9 @@
 import org.apache.kafka.connect.sink.SinkRecord;
 
 import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
 
 class KeyBuilder implements OutputFieldBuilder {
 
-    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
mental note; changing from plain object mapper to one with a exact big decimal. I don't think it may be relevant unless `record.key()` contains an object with big decimal values